gallery/src/components/NxTag/NxTagPage.tsx

```diff
@@ -130,6 +134,21 @@ const NxTagPage = () =>
       Light grey border added for clarity.
     </GalleryExampleTile>
 
+    <GalleryExampleTile title="NxTag With Tooltip"
+                        id="nx-tag-with-tooltip-example"
+                        liveExample={NxTagWithTooltipExample}
+                        codeExamples={NxTagWithTooltipExampleCode}>
+      An example of <NxCode>NxTag</NxCode> wrapped inside <NxCode>NxTooltip</NxCode>.{' '}
+      Notice that <NxCode>NxTooltip</NxCode> title overrides the tooltip generated when the text is truncated.
```

Comment:
> I think the NxTooltip title overrides both whether the text is truncated, right?

Ah I think I know what you mean here, it overrides overflowing tooltip.

Reply:
Yup! Oo I wonder if I should phrase it differently so it's more clear 🤔 I'll try to think of a better way to word it

Reply:
> > I think the NxTooltip title overrides both whether the text is truncated, right?
> 
> Ah I think I know what you mean here, it overrides overflowing tooltip.

I updated it here:
65b9f6875e9ebcc26539754a81dde17a5499de62
bf62e6d178a8a0a7407b97c9153ac961a7b1761e

Reply:
Further on this, I noticed in NxTag With Tooltip Example, if you remove the `title` attribute from `NxTooltip` overflowing tag, the `NxOverflowTooltip` is not showing. Is this expected?

Reply:
> Further on this, I noticed in NxTag With Tooltip Example, if you remove the `title` attribute from `NxTooltip` overflowing tag, the `NxOverflowTooltip` is not showing. Is this expected?

I believe so, because the `title` prop is required for NxTooltip